models/transaction.go

```diff
@@ -258,3 +273,12 @@ func ValidateTransaction(tx *gethTypes.Transaction) error {
 
 	return nil
 }
+
+func ValidateConsensusRules(
```

Comment:
why is defining this here needed? can't this just be called in the existing `ValidateTransaction`

Reply:
I can merge the two functions. I just wanted to keep them apart, so that they can be composed in other parts of the code-base. But maybe that was a premature thinking :+1:

Reply:
Function removed in https://github.com/onflow/flow-evm-gateway/pull/323/commits/cb82dc6d430f52b4831ba3ef895b8324d1a7cc2e